user-service/src/main/java/rs/edu/raf/banka1/bootstrap/BootstrapData.java

```diff
@@ -173,4 +196,115 @@ private Company createCompany() {
         return company;
     }
 
+    private static final Random random = new Random();
+
+    private LoanRequest generateLoanRequest() {
+        LoanRequest loanRequest = new LoanRequest();
+        loanRequest.setLoanType(generateRandomLoanType());
+        loanRequest.setLoanAmount(generateRandomLoanAmount());
+        loanRequest.setCurrency("RSD");
+        loanRequest.setLoanPurpose("Some purpose");
+        loanRequest.setMonthlyIncomeAmount(generateRandomIncomeAmount());
+        loanRequest.setMonthlyIncomeCurrency("RSD");
+        loanRequest.setPermanentEmployee(random.nextBoolean());
+        loanRequest.setEmploymentPeriod(generateRandomEmploymentPeriod());
+        loanRequest.setLoanTerm(generateRandomLoanTerm());
+        loanRequest.setBranchOffice("Branch");
+        loanRequest.setPhoneNumber("123456789");
+        loanRequest.setAccountNumber("123456789");
+        loanRequest.setStatus(LoanRequestStatus.PENDING);
+        return loanRequest;
+    }
+
+    private LoanRequest generateLoanRequest(final String curNum) {
+        LoanRequest loanRequest = new LoanRequest();
+        loanRequest.setLoanType(generateRandomLoanType());
+        loanRequest.setLoanAmount(generateRandomLoanAmount());
+        loanRequest.setCurrency("RSD");
+        loanRequest.setLoanPurpose("Some purpose");
+        loanRequest.setMonthlyIncomeAmount(generateRandomIncomeAmount());
+        loanRequest.setMonthlyIncomeCurrency("RSD");
+        loanRequest.setPermanentEmployee(random.nextBoolean());
```

Comment:
zasto ovo?

Reply:
da vidimo da se enum sacuva kao string a ne broj, al to cu naknadno, pa je zato seedovano